{
  "id": "EUVD-2021-2200",
  "enisaUuid": "001192e0-63b7-3b36-855a-f88596602e0e",
  "description": "An issue was discovered in OpenCV before 3.4.7 and 4.x before 4.1.1. There is an out of bounds read/write in the function HaarEvaluator::OptFeature::calc in modules/objdetect/src/cascadedetect.hpp, which leads to denial of service.",
  "datePublished": "Aug 1, 2019, 4:04:51 PM",
  "dateUpdated": "Aug 5, 2024, 12:19:41 AM",
  "baseScore": 7.5,
  "baseScoreVersion": "3.1",
  "baseScoreVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
  "references": [
    "https://nvd.nist.gov/vuln/detail/CVE-2019-14492",
    "https://github.com/opencv/opencv/issues/15124",
    "https://github.com/opencv/opencv-python",
    "https://github.com/opencv/opencv/compare/33b765d...4a7ca5a",
    "https://github.com/opencv/opencv/compare/371bba8...ddbd10c",
    "http://lists.opensuse.org/opensuse-security-announce/2019-12/msg00025.html",
    "https://www.suse.com/security/cve/CVE-2019-14492.html",
    "https://advisories.mageia.org/CVE-2019-14492.html",
    "https://ubuntu.com/security/CVE-2019-14492"
  ],
  "aliases": [
    "CVE-2019-14492",
    "GHSA-fw99-f933-rgh8"
  ],
  "assigner": "mitre",
  "epss": 0.53,
  "enisaIdProduct": [
    {
      "id": "c770cd37-bd07-3f5c-9e5b-71da11995302",
      "product": {
        "name": "n/a"
      },
      "product_version": "n/a"
    }
  ],
  "enisaIdVendor": [
    {
      "id": "9bb6d08a-d93c-3ec8-8e74-0c34a585a6a8",
      "vendor": {
        "name": "n/a"
      }
    }
  ],
  "cve": "CVE-2019-14492"
}